<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>840</width>
    <height>488</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTreeWidget" name="treeWidget">
      <property name="frameShape">
       <enum>QFrame::WinPanel</enum>
      </property>
      <property name="lineWidth">
       <number>2</number>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <column>
       <property name="text">
        <string>ID produkta</string>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Naziv produkta</string>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </column>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="pushButton_shraniNalepko">
      <property name="text">
       <string>Shrani nalepko</string>
      </property>
      <property name="icon">
       <iconset resource="icons.qrc">
        <normaloff>:/icons/save.ico</normaloff>:/icons/save.ico</iconset>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_nazivProdukta">
        <property name="minimumSize">
         <size>
          <width>115</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="frameShape">
         <enum>QFrame::WinPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="text">
         <string>Naziv produkta:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="buddy">
         <cstring>lineEdit_nazivProdukta</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_nazivProdukta"/>
      </item>
     </layout>
    </item>
    <item row="4" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_kolicina">
        <property name="minimumSize">
         <size>
          <width>115</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="frameShape">
         <enum>QFrame::WinPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="text">
         <string>Količina:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="buddy">
         <cstring>lineEdit_kolicina</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_kolicina"/>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_IDprodukta">
        <property name="minimumSize">
         <size>
          <width>115</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="frameShape">
         <enum>QFrame::WinPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="text">
         <string>ID produkta:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="buddy">
         <cstring>lineEdit_IDprodukta</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_IDprodukta"/>
      </item>
     </layout>
    </item>
    <item row="5" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_opombe">
        <property name="minimumSize">
         <size>
          <width>115</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::WinPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="lineWidth">
         <number>1</number>
        </property>
        <property name="text">
         <string>Opombe:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="buddy">
         <cstring>lineEdit_opombe</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_opombe"/>
      </item>
     </layout>
    </item>
    <item row="6" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="pushButton_izhod">
          <property name="text">
           <string>Izhod</string>
          </property>
          <property name="icon">
           <iconset resource="icons.qrc">
            <normaloff>:/icons/izhod.ico</normaloff>:/icons/izhod.ico</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_natisni">
          <property name="text">
           <string>Natisni nalepko</string>
          </property>
          <property name="icon">
           <iconset resource="icons.qrc">
            <normaloff>:/icons/print.ico</normaloff>:/icons/print.ico</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>840</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMeni">
    <property name="title">
     <string>Meni</string>
    </property>
    <addaction name="actionNov_napis"/>
    <addaction name="actionPrint"/>
    <addaction name="actionShrani_nalepko"/>
    <addaction name="separator"/>
    <addaction name="actionIzhod"/>
   </widget>
   <widget class="QMenu" name="menuO_progrramu">
    <property name="title">
     <string>O programu</string>
    </property>
    <addaction name="actionO_programu"/>
   </widget>
   <addaction name="menuMeni"/>
   <addaction name="menuO_progrramu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionPrint">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icon.ico</normaloff>:/icons/icon.ico</iconset>
   </property>
   <property name="text">
    <string>Natisni nalepko...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionIzhod">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/izhod.ico</normaloff>:/icons/izhod.ico</iconset>
   </property>
   <property name="text">
    <string>Izhod</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="text">
    <string>Edit</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>Izbriši nalepko</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Izbriše nalepko&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
  <action name="actionO_programu">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/about.ico</normaloff>:/icons/about.ico</iconset>
   </property>
   <property name="text">
    <string>O programu...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionShrani_nalepko">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/save.ico</normaloff>:/icons/save.ico</iconset>
   </property>
   <property name="text">
    <string>Shrani nalepko...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionNov_napis">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/new.ico</normaloff>:/icons/new.ico</iconset>
   </property>
   <property name="text">
    <string>Nov napis...</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>lineEdit_IDprodukta</tabstop>
  <tabstop>lineEdit_nazivProdukta</tabstop>
  <tabstop>lineEdit_kolicina</tabstop>
  <tabstop>lineEdit_opombe</tabstop>
  <tabstop>pushButton_natisni</tabstop>
  <tabstop>pushButton_shraniNalepko</tabstop>
  <tabstop>treeWidget</tabstop>
  <tabstop>pushButton_izhod</tabstop>
 </tabstops>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_izhod</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>408</x>
     <y>440</y>
    </hint>
    <hint type="destinationlabel">
     <x>522</x>
     <y>448</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
